cucumber_task/features/step_definitions/task.rb

```diff
@@ -0,0 +1,110 @@
+# Add config file to the project
+require_relative 'config'
+
+# Constants
+page_obj = Page.new
+patients = page_obj.patient_id
+
+# Test execution
+browser.get(MAIN_PAGE)
+
+Given('patient page is opened') do
+  page_obj.login('admin', 'Admin123')
+end
+
+
+When('opening patient page') do
+  page_obj.find_patient_btn.click
+end
+
+Then('patient table is not empty') do
+  PATIENT_INFO.each{|text| !expect(main_pg.columm(text)) == ''}
+  page_obj.home_btn.click
+end
+
+
+When('register patient is opened and all data fields are filled') do
+  page_obj.reg_process(PATIENT_NAME, PATIENT_DATE, GENDER[0], MONTHS[0])
```

Comment:
In this field we can change index

e.g GENDER[0] = Male, GENDER[1] = Female

MONTHS[0] = January ... MONTHS[11] = December